astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -357,7 +357,7 @@ def register(cls, name: str, parameters: dict, references=None,
         The ICRS coordinates of the Galactic center.
     galcen_distance : `~astropy.units.Quantity`, optional, keyword-only
         The distance from the sun to the Galactic center.
-    galcen_v_sun : `~astropy.coordinates.representation.CartesianDifferential`, `~astropy.units.Quantity` ['speed'], optional, keyword-only
+    galcen_v_sun : `~astropy.coordinates.representation.cartesian.CartesianDifferential`, `~astropy.units.Quantity` ['speed'], optional, keyword-only
```

Comment:
Hmm, none of these changes should be necessary (after all, we can use `~astropy.units.Quantity`), but perhaps the sphinx documentation settings are not correct? Or would just `~astropy.coordinates.CartesianDifferential` work?

Reply:
A good point. I'll try a TMP push with it reverted.

Reply:
I've had success with ``astropy.coordinates...CartesianDifferential``